app/models/concerns/image_file.rb

```diff
@@ -19,6 +19,21 @@ module ImageFile
     enum status: [ :started, :created, :processing, :complete, :error, :retrying, :cancelled ]
   end
 
+  class_methods do
+    def build(file)
```

Comment:
Helper to take in either a string url, a hash, or a model - normalize that and return either a `WhatevImage` model or nil if no original_url was set.

Used by the Feed/Episode setter methods, to find-or-build images.